crates/bevy_ptr/src/lib.rs

```diff
@@ -599,7 +599,7 @@ trait DebugEnsureAligned {
 impl<T: Sized> DebugEnsureAligned for *mut T {
     #[track_caller]
     fn debug_ensure_aligned(self) -> Self {
-        let align = core::mem::align_of::<T>();
+        let align = align_of::<T>();
```

Comment:
This one is a false positive I think; core::mem isn't imported here as far as I know, so this needs qualification. It's why the CI failed.

Reply:
The question is… why? Not only the [std prelude](https://doc.rust-lang.org/std/prelude/index.html), also the [core prelude include `align_of`](https://doc.rust-lang.org/core/prelude/v1/index.html) per default and the stable (non-MSRV) checks do not notice this. What's different between MSRV and stable? I highly doubt that the prelude changed between 1.79 which is the current bevy MSRV and 1.80 which is the current stable, so the error must be somewhere else.

Reply:
Actually, 1.80 adds `align_of` to the prelude. I wouldn't have expected such a change and expected it to be tied to an edition. Which isn't the case.

Reply:
I added them explicitly to the use statements in 40237e1675866c44890e5d67411502b2c81834fa (and plan on open an issue about this in the rust repo). Interestingly `unused_imports` (warn by default) doesn't seem to care about this on 1.80.

Reply:
https://github.com/rust-lang/rust/issues/129306